desktop/package.json

```diff
@@ -20,6 +20,7 @@
     "postinstall": "babel-node --presets es2015,stage-2 npm-helper.js postinstall",
     "render-screenshots": "babel-node --presets es2015,stage-2 npm-helper.js render-screenshots",
     "updated-fonts": "babel-node --presets es2015,stage-2 npm-helper.js updated-fonts",
+    "wrap": "rm -rf node_modules/msgpack && npm shrinkwrap --dev",
```

Comment:
Would `npm prune` work as an alternative to the `rm`? That way if you have any other extra packages hanging around they get cleaned up too.